libs/docs/src/docs/no-version/developing-with-devfiles.md

```diff
@@ -3,78 +3,44 @@ title: Developing with devfiles
 description: Developing with devfiles
 ---
 
-Use the devfile specification to develop a Node.js “Hello World”
-application. Developing this application introduces you to how a devfile
-automates and simplifies your development process.
-
-## Prerequisites
-
-To make developing with the devfile specification easier, consider doing
-the following:
+To get a better understanding of what devfiles can help build, take a look at a few tools that currently support devfile.
 
-- Install [minikube](https://minikube.sigs.k8s.io/docs/start/) to
-  create a Kubernetes cluster on your local machine.
+## Quick start guides
 
-- Download
-  [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-macos/)
-  to access your namespace.
-
-- Enable [ingress on
-  minikube](https://kubernetes.io/docs/tasks/access-application-cluster/ingress-minikube/)
-  to access your devfile project.
+- Use the devfile specification to develop a Node.js “Hello World”
+application using `odo`. Developing this application introduces you to how a devfile
+automates and simplifies your development process.
 
-- Install the [odo command line interface (CLI)
-  tool](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.6/html/cli_tools/developer-cli-odo#installing-odo)
-  to execute the devfile specification.
+Get started with [odo](https://odo.dev/docs/user-guides/quickstart/nodejs).
 
-## Procedure
+- Create a workspace in `Eclipse Che` with a community sample backed by the devfile specification to start building your application in the language of your choice.
 
-1. View the available devfiles:
+Get started with [Eclipse Che](https://www.eclipse.org/che/).
 
-    ```shell-session
-    $ odo catalog list components
-    ```
+- Use blueprints in `Amazon CodeCatalyst` to quickly build a Modern three-tier web application. Start working on the source code with a Dev Environment that uses a devfile to pre-determine and installs the required project tools and application libraries.
```

Comment:
- Let's quote "Modern three-tier web application" because that's the name of the sample.   e.g.

> Use blueprints in `Amazon CodeCatalyst` to quickly build the "Modern three-tier web applicaton" sample

- "uses a devfile to pre-determine and installs"  - de-pluralize "installs"